http-client-core/src/main/java/io/micronaut/http/client/HttpClientConfiguration.java

```diff
@@ -833,6 +865,69 @@ public void setAddressResolverGroupName(@Nullable String addressResolverGroupNam
         this.addressResolverGroupName = addressResolverGroupName;
     }
 
+    /**
+     * For HTTP/2 connections, the interval from the last inbound message to when an automated ping
+     * should be sent. This can be used to keep low-traffic connections alive.
+     *
+     * @return The timeout when to send a ping frame
+     */
+    @Nullable
+    public Duration getHttp2PingIntervalRead() {
```

Comment:
Would it make sense to extract those into some http2 config? The property would be `http2.pingIntervalRead`

Reply:
i think then we'd have to duplicate it for the different HttpClientConfiguration classes too, im not sure how the config api works there

Reply:
I agree with @dstepanov this would be better

Reply:
ok i will try it